### ISSUE TITLE
Cannot reproduce the performance in paper?

### ISSUE THREAD

**Issue by XuRuihan** [HJYao00/DenseConnector#12]
I'm interested in your work but I have run into problems reproducing your results. With the given code and training scripts (`scripts/v1_5/pretrain.sh` and `scripts/v1_5/finetune.sh`), I can only get a GQA accuracy of 63.08, which is far lower than 63.8 reported in the paper. Am I missing some important parts?

And by the way, the eval scripts cannot run successfully. I'm not sure where is the model path `DenseConnector-v1.5-7B`, so I use the `checkpoints_stage2/DenseConnector-v1.5-7b-FineTuning` path. Is there a problem of this operation?
